html-css/week1/jamshedali/assignment-week-1/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <!--header start -->
+    <header>
+        <h1>Order Summary</h1>
+        <div class="summary_total">
+            <div>Total</div>
+            <div>$333/month</div>
+        </div>
```

Comment:
Good class names used, however, you can add a bit more semantics to the markup. Avoid wrapping text simply in a `<div>` it can always be either `<p>`, `<h1-6>` or `<span>`. Here best would work this:
```suggestion
        <p class="summary_total">
            <span>Total</span>
            <span>$333/month</span>
        </p>
```

Reply:
<p class="summary_total">
            <span>Total</span>
            <span>$333/month</span>
        </p>
        <p class="grand_total">
            <span>Total</span>
            <span>$100</span>
        </p> done

Reply:
i am also checking css about changing effects on index.html